playbooks/byo/openshift-checks/health.yml

```diff
@@ -1,3 +1,10 @@
 ---
 - include: ../openshift-cluster/initialize_groups.yml
+  tags:
+  - always
```

Comment:
Agreed.  We need to either fix and test the usage of tags in the repo or we need to remove all usage of tags.  I'll bring it up on the Arch Meeting for consideration.